platform/mellanox/asic_table.j2

```diff
@@ -40,6 +40,7 @@
         'x86_64-mlnx_msn4700-r0':'MELLANOX-SPECTRUM-3',
         'x86_64-mlnx_msn4600c-r0':'MELLANOX-SPECTRUM-3',
         'x86_64-mlnx_msn4600-r0':'MELLANOX-SPECTRUM-3',
+        'x86_64-nvidia_sn2201-r0': 'MELLANOX-SPECTRUM',
```

Comment:
>  [](http://example.com/codeflow?start=34&length=1)

Minor issue: remove extra blank

Reply:
I couldn't find any whitespace to remove, am I missing something?

Reply:
@qiluo-msft I removed the whitespace.